taxi/forms.py

```diff
@@ -0,0 +1,54 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverUpdateForm(forms.ModelForm):
```

Comment:
you have to create 2 forms: for creating the driver and for updating license_number. In both forms, you must validate license_number